server/src/utils/bb2Util.py

```diff
@@ -64,6 +67,7 @@ def getBenefitData(settings, configsSettings, query, loggedInUser):
     }
     BB2_BENEFIT_URL = configsSettings.get('bb2BaseUrl') + '/' + settings.version + '/fhir/ExplanationOfBenefit/'
     myHeader = {'Authorization' : 'Bearer '+loggedInUser.get('authToken').get('access_token')}
-    beneResponse = requests.get(url=BB2_BENEFIT_URL,params=PARAMS,headers=myHeader)    
+    beneResponse = requests.get(url=BB2_BENEFIT_URL,params=PARAMS,headers=myHeader)
```

Comment:
Is the retry logic baked into the requests library? If so, do we need to remove the retry from the authorization endpoints? So far I'm basically just seeing added logging (which I'm not actually sure we want here)

Reply:
haven't put in the retry code yet, working on it.

Reply:
error response reporting + retry committed.